blueocean-git-pipeline/src/main/java/io/jenkins/blueocean/blueocean_git_pipeline/GitPipelineUpdateRequest.java

```diff
@@ -0,0 +1,137 @@
+package io.jenkins.blueocean.blueocean_git_pipeline;
+
+import com.cloudbees.plugins.credentials.common.StandardCredentials;
+import hudson.model.Cause;
+import hudson.model.CauseAction;
+import hudson.model.Item;
+import hudson.security.ACL;
+import hudson.util.PersistedList;
+import io.jenkins.blueocean.commons.ErrorMessage;
+import io.jenkins.blueocean.commons.ServiceException;
+import io.jenkins.blueocean.rest.Reachable;
+import io.jenkins.blueocean.rest.model.BluePipeline;
+import io.jenkins.blueocean.rest.model.BluePipelineCreateRequest;
+import io.jenkins.blueocean.rest.model.BlueScmConfig;
+import jenkins.branch.BranchSource;
+import jenkins.branch.MultiBranchProject;
+import jenkins.model.Jenkins;
+import jenkins.plugins.git.GitSCMSource;
+import org.acegisecurity.Authentication;
+import org.apache.commons.lang.StringUtils;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.annotation.CheckForNull;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * @author Vivek Pandey
+ */
+public class GitPipelineUpdateRequest extends BluePipelineCreateRequest {
```

Comment:
No, we just need the create request, it's a byproduct of the merge I didn't want to remove until I can have it all working without it.